repos/system_upgrade/common/actors/insightsautoregister/libraries/insightsautoregister.py

```diff
@@ -0,0 +1,25 @@
+from leapp.libraries.common import rhsm
+from leapp.libraries.common.config import get_env
+from leapp.libraries.stdlib import api, CalledProcessError, run
+
+
+def _insights_register():
+    try:
+        run(['insights-client', '--register'])
+        api.current_logger().info('Automatically registered into Red Hat Insights')
+    except (CalledProcessError, OSError) as err:
```

Comment:
Can this actually produce OSError?

Reply:
The `insights-client` package might fail to install. I think I actually need FileNotFoundError in this case

Reply:
That should be handled by `run` actually. I will remove the OSError